sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/SqlDelightDatabase.kt

```diff
@@ -259,11 +244,11 @@ abstract class SqlDelightDatabase @Inject constructor(
         it.description = "Verify ${source.name} $name migrations and CREATE statements match."
         it.properties = getProperties()
         it.verifyMigrations = verifyMigrations.get()
-        it.classpath.setFrom(configuration.fileCollection { true })
-        it.classpath.from(moduleConfiguration.fileCollection { true })
```

Comment:
i swear there was a reason i had to do this that was really dumb, should have commented